### ISSUE TITLE
[Icon] Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.

### ISSUE THREAD

**Issue by zbeedatm** [Semantic-Org/Semantic-UI#6854]
### Steps
use:

> < a href="#">
             < Icon link name="pause circle outline" size="large"/>
    < /a>

### Expected Result
No warning should be displayed!

### Actual Result
getting:
Warning: Failed prop type: Invalid prop `aria-hidden` of type `boolean` supplied to `Icon`, expected `string`.

### Version
"semantic-ui-react": "^0.87.3"

### Testcase
[Fork, update, and replace to show the bug]:
https://jsfiddle.net/ca0rovs3/

**Comment by zbeedatm** [comment]
Please ignore
